versions.props

```diff
@@ -11,7 +11,7 @@ com.google.guava:guava = 23.6.1-jre
 com.google.guava:guava-testlib = 23.6.1-jre
 com.palantir.remoting3:* = 3.46.0
 com.palantir.safe-logging:* = 1.9.0
-com.palantir.tracing:* = 2.13.1
+com.palantir.tracing:* = 2.14.0
```

Comment:
@schlosna Thoughts on tagging a release with this change? Tritium is one of our most widely used libraries with tracing among non-wc services, I imagine this transitive dependency bump will reduce trace spam in some places.

Reply:
tagged a 0.12.4 with this + the Jackson 2.9.9

Reply:
Thanks!